src/postgraphql/http/createPostGraphQLHttpRequestHandler.d.ts

```diff
@@ -22,6 +22,8 @@ export default function createPostGraphQLHttpRequestHandler (config: {
   // The actual GraphQL schema we will use.
   getGqlSchema: () => Promise<GraphQLSchema>,
 
+  emitter: any,
```

Comment:
I also feel like there is a better solution, for now could you rename to `_emitter` or throw a `TODO` comment above it. This is definitely the best solution for now, but in the future when we add subscriptions we may have a better story for events.